Formula/clang-build-analyzer.rb

```diff
@@ -0,0 +1,35 @@
+class ClangBuildAnalyzer < Formula
+  desc "Tool to analyze compilation time"
+  homepage "https://github.com/aras-p/ClangBuildAnalyzer"
+  url "https://github.com/aras-p/ClangBuildAnalyzer/archive/v1.4.0.tar.gz"
+  sha256 "dae8e7838145a72c01c397c3998d9f6801fc4dc819d552010d702cab7dede530"
+  license "Unlicense"
+  head "https://github.com/aras-p/ClangBuildAnalyzer.git", branch: "main"
+
+  depends_on "cmake" => :build
+  depends_on :macos
+
+  def install
+    system "cmake", "-S", ".", "-B", "build", *std_cmake_args
+    system "make", "-C", "build"
+    bin.install ["build/ClangBuildAnalyzer"]
```

Comment:
Why aren't we using an `install` target?

Reply:
Fixed